openwisp_controller/connection/connectors/ssh.py

```diff
@@ -163,15 +163,14 @@ def exec_command(
         # log standard error
         error = stderr.read().decode('utf-8', 'ignore')
         if error:
-            logger.error(error)
             if not output.endswith('\n'):
                 output += '\n'
             output += error
         # abort the operation if any of the command
         # returned with a non-zero exit status
         if exit_status not in exit_codes and raise_unexpected_exit:
             log_message = 'Unexpected exit code: {0}'.format(exit_status)
-            logger.error(log_message)
+            logger.info(log_message)
```

Comment:
Logging log_message here is only useful when output is empty. We should do it on info level since an exception will be ultimately raised by the following code.